src/FakeItEasy/Creation/CastleDynamicProxy/CastleDynamicProxyInterceptionValidator.cs

```diff
@@ -8,15 +8,24 @@ namespace FakeItEasy.Creation.CastleDynamicProxy
 
     internal class CastleDynamicProxyInterceptionValidator : IMethodInterceptionValidator
     {
+        private const string NonVirtualMethodsCantBeIntercepted =
+            "Non-virtual or sealed members can not be intercepted. Only interface members and non-sealed virtual, overriding, and abstract members can be intercepted.";

```

Comment:
We were already using this message in two places, and I added a third, so it was probably time to introduce a constant.

I reworded the message to also mention sealed members, which can never be intercepted.